multimedia/scrcpy/Portfile

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        Genymobile scrcpy 1.17 v
+github.tarball_from releases
+revision            0
+
+categories          multimedia
+platforms           darwin
+license             Apache-2
+maintainers         {bochtler.io:macports @MarcelBochtler} \
+                    openmaintainer
+description         Display and control your Android device
+long_description    This application provides display and control of Android \
+                    devices connected on USB (or over TCP/IP). It does not \
+                    require any root access.
+
+distfiles-append scrcpy-server-v${version}
+
+checksums           ${distname}${extract.suffix} \
+                    rmd160  b0909f342a61e6001fe1643df98f38d0dcc99641 \
+                    sha256  b3f5490970dbef6f50725b4d46ba07baed9c7eb19696e71029257a01044c6664 \
+                    size    266684 \
+                    scrcpy-server-v${version} \
+                    rmd160  59bc2a451016f5c5f11a22643a441a19e3a78b9b \
+                    sha256  11b5ad2d1bc9b9730fb7254a78efd71a8ff46b1938ff468e47a21b653a1b6725 \
+                    size    34930
+
+depends_build-append \
+                    port:meson \
+                    port:pkgconfig
+
+depends_lib-append  path:lib/libavcodec.dylib:ffmpeg \
+                    port:libsdl2
+
+configure.cmd       meson x --buildtype release --strip -Db_lto=true -Dprebuilt_server=${distpath}/scrcpy-server-v${version}
+
+build.cmd           ninja -Cx
+
+destroot {
+    xinstall -m 755 -d ${destroot}${prefix}/share/${name}
+    xinstall -m 644 ${worksrcpath}/x/server/scrcpy-server ${destroot}${prefix}/share/${name}
+
+    xinstall -m 755 ${worksrcpath}/x/app/scrcpy ${destroot}${prefix}/bin
+}
+
+notes "
+${name} requires adb. It can be installed using port install android-platform-tools.
+"
```

Comment:
Seriously looking forward to this. Thanks @MarcelBochtler!

Can `android-platform-tools` be set in `depends_run`, similar to ports such as [logcat-color3](https://github.com/macports/macports-ports/tree/master/python/logcat-color3/Portfile)?

In that port they use the following, which should hopefully also be applicable here:

```tcl
depends_run-append \
                bin:adb:android-platform-tools
```

Reply:
Yes, that would be much better.

Reply:
Done